zips/draft-ecc-community-and-coinholder.md

```diff
@@ -0,0 +1,188 @@
+
+This model would be activated for 3 years, allowing enough time to determine whether it should be changed or codified for longer.
+
+
+# Motivation
+
+If no action is taken, in November 2025 funds from block subsidies will stop being directed to Zcash Community Grants [^zip-1015-zcg] and to the Deferred Dev Fund Lockbox established by ZIP 2001 [^zip-2001]. If the block subsidies stop, it will risk a gap in funding of Zcash development organisations, either via ZCG grants or via potential future disbursements from the Deferred Dev Fund Lockbox.
+
+This proposal aims to:
+* decentralize decision-making,
+* hold stakeholders accountable for outcomes,
+* be dynamic enough to allow for change, and
+* provide clarity on decision-making.
+
+It would immediately increase coin-holders’ voice, minimize governance confusion, and simplify decision-making.
+
+
+# Requirements
+
+* There is a well-defined, publically agreed, process for evaluation and community feedback on grant proposals.
+* Funds are held in a multisig resistant to compromise of some of the parties' keys, up to a given threshold.
+* No single party's non-cooperation or loss of keys is able to cause any Protocol-defined Ecosystem Funding to be locked up unrecoverably.
+* During the period of this proposal, the block rewards must be distributed as described in the [Abstract] above.
+* The funds from the 8% and 12% funding streams will be usable immediately.
+* The funds in the Deferred Dev Fund Lockbox will be usable immediately.
+* Any use of Deferred Dev Fund Lockbox funds is consistent with the purpose specified in [^zip-1015-lockbox] of "funding grants to ecosystem participants".
+
+
+# Non-requirements
+
+Any changes to the ZCG or its governance, such as its expansion to include more members, may be desired but are specifically outside this proposal’s scope.
+
+
+# Specification
+
+This proposal empowers both the community (through ZCG) and coin holders to independently determine what, if anything, should be funded through development fund grants.
+
+The funding streams described below will be defined in a new revision of ZIP 214 [^zip-0214].
+
+## Zcash Community Grants
+
+A funding stream will be established for Zcash Community Grants, consisting of 8% of the block subsidy, and subject to all of the same rules currently specified in ZIP 1015 [^zip-1015-zcg].
+
+This funding stream will start on expiry of the existing ``FS_FPF_ZCG`` funding stream [^zip-1015-funding-streams], and last for a further 1,260,000 blocks (approximately 3 years), ending at Zcash's 3rd halving.
+
+## Coin-holder-controlled Fund
+
+### One-time lockbox disbursal
+
+The coinbase transaction of the activation block of this ZIP MUST include an additional output to a 3-of-5 P2SH multisig with keys held by the Zcash Foundation, the Electric Coin Company, Shielded Labs, and two other organizations yet to be decided ("Key-holder Organizations").
```

Comment:
nit: I think that `Key-Holder`, `Coin-Holder`, and `Coin-Holder Controlled` should have all words capitalized; to capitalize just the first word of the hyphenated phrase looks wrong to me.
```suggestion
The coinbase transaction of the activation block of this ZIP MUST include an additional output to a 3-of-5 P2SH multisig with keys held by the Zcash Foundation, the Electric Coin Company, Shielded Labs, and two other organizations yet to be decided ("Key-Holder Organizations").
```

If accepted, apply this suggestion throughout.